extensions/smallrye-graphql-client/runtime/src/main/java/io/quarkus/smallrye/graphql/client/runtime/GraphQLClientConfig.java

```diff
@@ -130,4 +161,10 @@ public class GraphQLClientConfig {
      */
     @ConfigItem
     public Optional<Boolean> allowUnexpectedResponseFields;
+
+    /**
+     * The name of the TLS configuration (bucket) used for client authentication in the TLS registry.
+     */
+    @ConfigItem
+    public Optional<String> tlsConfigurationName;
```

Comment:
Just to make sure, this is consistent with the other extensions, right?

Reply:
Yeah, we copied this from the gRPC client

Reply:
Yes, I confirm this is the name used everywhere and documented in the ADR.